src/network/clientpackethandler.cpp

```diff
@@ -122,7 +123,12 @@ void Client::handleCommand_AuthAccept(NetworkPacket* pkt)
 					<< m_recommended_send_interval<<std::endl;
 
 	// Reply to server
-	NetworkPacket resp_pkt(TOSERVER_INIT2, 0);
+	std::string lang = gettext("LANG_CODE");
+	if (lang == "LANG_CODE")
+		lang = "";
+	lang = "fr";
```

Comment:
Yes, there is one commit that is there only for testing and that needs to be removed before this is merged.